src/test/datascience/notebook/standard_widgets.ipynb

```diff
@@ -156,15 +156,6 @@
  "metadata": {
   "celltoolbar": "Attachments",
   "file_extension": ".py",
-  "kernelspec": {
```

Comment:
I was having some issues with getting the right kernelspec in the test, so I removed this, so it would just rely on the language info. I believe that I could put this back if desired.

Reply:
I noticed that a number of ipynbs that we use in our tests have some local machine specific info in them. Might be worth scrubbing that at some point.

Reply:
Oh, thanks.
In some tests (perhaps one), i remove the metadata.